blockchain/blockchain_test.go

```diff
@@ -448,7 +448,7 @@ func TestEvents(t *testing.T) {
 
 	from := utils.HexToFelt(t, "0x49d36570d4e46f48e99674bd3fcc84644ddd6b96f7c741b1562b82f9e004dc7")
 	t.Run("filter with no keys", func(t *testing.T) {
-		filter, err := chain.EventFilter(from, []*felt.Felt{})
+		filter, err := chain.EventFilter(from, [][]felt.Felt{{}, {}, {}})
```

Comment:
Why is the size of slice 3? Can you not just have `[][]felt.Felt{{}}` to get all the keys?